.travis.yml

```diff
@@ -1,30 +1,31 @@
 languages: java
 # TODO: Add openjdk8 once it is available in Travis.
 jdk:
-  - oraclejdk8
+- oraclejdk8
 sudo: false
-script: ./gradlew clean build coveralls --scan --no-daemon --info
+script: ./gradlew clean build --scan --no-daemon --info
 
 # see https://docs.travis-ci.com/user/languages/java/#Projects-Using-Gradle
 before_cache:
-  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
-  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+- rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
```

Comment:
I suggest keeping formatting changes a separate commit.

Reply:
This is what happens when IDE is trying to be too smart... The formatting/whitespace changes are courtesy of IntelliJ. Sorry about that, I can try and undo or isolate them.